recipes/apriltag/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "apriltag" %}
+{% set version = "3.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/AprilRobotics/apriltag/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 68ef6640f55aac6566ba8d030a4e7d960446d5b9340dfb61e79b9f92a08b1032
+
+build:
+  number: 0
+  script: |
+    cmake -G Ninja -B build -D CMAKE_INSTALL_PREFIX:PATH=$PREFIX -D BUILD_SHARED_LIBS=ON -D BUILD_PYTHON_WRAPPER=OFF
+    cmake --build build --target install
+
+requirements:
+  build:
+    - compilers
```

Comment:
Please use the Jinja functions. The compilers package should not be used in a recipe.